src/tss2-tcti/tcti-cmd.c

```diff
@@ -622,7 +622,7 @@ TSS2_RC Tss2_Tcti_Cmd_Init (TSS2_TCTI_CONTEXT *tctiContext, size_t *size,
 }
 
 /* public info structure */
-const TSS2_TCTI_INFO tss2_tcti_info = {
+const TSS2_TCTI_INFO tss2_tcti_cmd_info = {
```

Comment:
@AndreasFuchsTPM any reason not to make these `static` too, just to reduce the incidence of these?